cl/search/fixtures/florida_state_court_data.json

```diff
@@ -0,0 +1,3726 @@
+[
+  {
+    "fields": {
+      "jurisdiction": "ST",
+      "position": 355.05,
+      "short_name": "",
+      "full_name": "Florida Judge's Courts",
+      "citation_string": "",
+      "end_date": "1972-12-31",
+      "in_use": false,
+      "url": "",
+      "date_modified": "2021-09-01T21:54:53.115Z",
+      "has_oral_argument_scraper": false,
+      "has_opinion_scraper": false,
+      "start_date": null,
+      "notes": "Created as part of import of Case.Law data."
+    },
+    "model": "search.court",
+    "pk": "flajudct"
+  },
+  {
+    "fields": {
+      "jurisdiction": "ST",
+      "position": 355.0501,
+      "short_name": "Broward County Judges' Court",
```

Comment:
Or is this apostrophe wrong? :)

Reply:
This one is wrong and they have been made uniform